mail/postfix/Makefile

```diff
@@ -10,8 +10,8 @@ include $(TOPDIR)/rules.mk
 PKG_NAME:=postfix
 PKG_RELEASE:=3
 PKG_SOURCE_URL:=ftp://ftp.porcupine.org/mirrors/postfix-release/official/
```

Comment:
Preferably add a https/http site/mirror in front of the ftp one.
http://www.postfix.org/download.html

Reply:
As far as I understand, it will be cached on openwrt/lede servers anyway. Besides that, how do we prefer one mirror over another? Is there any best practices?

Reply:
The mirrors server(s) are not reliable as they for whatever reason don't host all tarballs and both are on the same network (Hetzner) so redundancy is poor. As far choosing mirrors goes as long as it has good peering for Europe/North America (as the majority of users are located in those areas).  

Having a quick look this should be fine: https://high5.nl/mirrors/postfix-release/index.html
Most US/NA mirrors are either dead or ftp.

Another (easier/better solution) would be having this in the LEDE tree (imho): 
https://github.com/lede-project/source/pull/1133